zkit/src/main.rs

```diff
@@ -108,7 +109,9 @@ struct VerifyOpt {
 struct GenerateVerifierOpt {
     #[arg(short, default_value = "vk.bin")]
     vk_file: String,
-    #[arg(long = "s", default_value = "verifier.sol")]
+    #[arg(short, default_value = "groth16")]
```

Comment:
Better to keep plonk as the default value

Reply:
done